SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -11385,30 +11391,24 @@ void CEulerSolver::BC_ActDisk_VariableLoad(CGeometry *geometry, CSolver **solver
   su2double Pressure_out, Density_out,
   Pressure_in, Density_in;
 
-  su2double C[MAXNDIM], Prop_Axis[MAXNDIM], R, r[MAXNDIM], r_;
+  su2double Prop_Axis[MAXNDIM];
```

Comment:
Thanks, these were fixed in #1044 already, along with the indentation of the function.